compile.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if ! command -v llama &> /dev/null
+then
+    cd /usr/src/app
+    git clone https://github.com/ggerganov/llama.cpp.git --branch master-d5850c5
+    cd llama.cpp
+    make 
+    mv main /usr/bin/llama
```

Comment:
Add cmd to remove the src files after the `git clone`

Reply:
I feel like the `git clone` should happen during the CI Build process, and the `make` part during runtime.

Else the container can't be used in non-internet connected environments.

Reply:
Good catch thanks, it's fixed now